plugins/turing/go.mod

```diff
@@ -4,35 +4,35 @@ go 1.18
 
 require (
 	bou.ke/monkey v1.0.2
+	github.com/caraml-dev/mlp v1.7.6
```

Comment:
Shall we use the same version of the MLP API as the management and treatment services, for the plugin too?

Reply:
Actually, I think it makes more sense to update management and treatment services to be `v1.7.6`, wdyt?